### DISCUSSION TITLE
Move from WPF to Avalonia

### DISCUSSION THREAD

**Discussion by buchwasa** [git-ecosystem/git-credential-manager#689] [Q&A] [answered]
Are there any plans to move from WPF to Avalonia? I was looking into [issue 682](https://github.com/GitCredentialManager/git-credential-manager/issues/682) and was looking at doing it myself, but I think in the current state of Windows being WPF and everything else being Avalonia makes it more of a hassle to do.

If there is a plan to eventually transition over to Avalonia but nothing for that has been done yet, I'd be happy to take it on.

**Comment by ldennington** [comment, marked as answer]
Hey @buchwasa -

TLDR, we stuck with the WPF UI on Avalonia because we're still tied to .NET Framework, and making that work with Avalonia is a bit tricky. So I think we're going to continue to punt on this until we are able to fully cut ties with .NET Framework (check out [this issue](https://github.com/GitCredentialManager/git-credential-manager/issues/113) for details on the current blockers).
